index.html

```diff
@@ -136,24 +136,26 @@ <h2>Contact Me</h2>
                 If you have an application you are interested in developing, <br /> a feature that you need built or a project <br /> 
                  that needs coding. I’d love to help with it 
               </p>
-              <form name="CONTACT" class="form-controls">
+              <form name="form" class="form-controls" id="form" action="https://formspree.io/f/mwkyzddk"
+              method="POST">
                 <label for="name-field">&nbsp;</label>
                 <input type="text" id="name-field" name="NAME" placeholder="Esi Ogagaoghene"  aria-label="Name" tabindex="4" required/>
                 <div class="break"></div>
                 <label for="email-field">&nbsp;</label>
-                <input type="email" id="email-field" name="EMAIL" placeholder="esi.ogagaoghene@yahoo.com" tabindex="5" aria-label="Email" required/>
+                <input type="email" id="email" name="email" placeholder="esi.ogagaoghene@yahoo.com" tabindex="5" aria-label="Email" required/>
                 <div class="break"></div>
                 <label for="comment-field">&nbsp;</label>
                 <textarea rows="4" cols="10" id="comment-field" name="COMMENT" placeholder="Write your message" tabindex="6" aria-label="C" required></textarea>
               </form>
-              <button type="submit" name="bttn-submit" class="form-button" tabindex="7">Get In Touch</button>
+              <button type="submit" name="submit" id="submit" class="form-button" tabindex="7">Get In Touch</button>
```

Comment:
- [ ] You are almost there! However, your form is not submitting the data, this is happening because the button is outside the form, there's a way to tie it to your form but you will need to reference it, it would be great to fix this issue.

Reply:
Sure it will be great to fix that issue. I have resolved the issue by hooking up the button to the form with the attribute "form" placed in the button. e.g <button form="name_of_form">, and it is working fine.  I got the message below after a successful submission. 
Thanks!

The form was submitted successfully.
Go back